### ISSUE TITLE
Making timeout units consistent

### ISSUE THREAD

**Issue by wangvsa** [llnl/UnifyFS#811]
Current configuration allows setting three timeouts, but they don't use the same unit.

margo section:
- client_timeout: timeout in milliseconds for rpcs between client and server (default: 5000)
- server_timeout: timeout in milliseconds for rpcs between servers (default: 15000)

server section:
- init_timeout: timeout in seconds to wait for servers to be ready for clients (default: 120)

Maybe better to make the three timeout units the same?  Probably all use seconds. 
Otherwise, It can be confusing when checking a job script with all three environment variables:
```bash
export UNIFYFS_SERVER_INIT_TIMEOUT=600
export UNIFYFS_MARGO_CLIENT_TIMEOUT=60000
export UNIFYFS_MARGO_SERVER_TIMEOUT=60000
```

**Comment by MichaelBrim** [comment]
I agree that switching to use seconds uniformly would be preferred, since I don't see a good reason to use sub-second RPC timeouts
